dvc/remote/local/__init__.py

```diff
@@ -119,10 +129,9 @@ def exists(self, path_info):
         return os.path.lexists(fspath_py35(path_info))
 
     def makedirs(self, path_info):
-        if not self.exists(path_info):
-            os.makedirs(fspath_py35(path_info))
+        makedirs(path_info, exist_ok=True, mode=self._dir_mode)
```

Comment:
:thinking: a little bit confusing to use a method with the same name under the method, what do you think about using `utils.makedirs`? or import it differently?

Reply:
But this is `makedirs` and `RemoteLOCAL.makedirs`(or `self.makedirs`), so it doesn't create collisions. If we were to change this one to `utils.makedirs`, then we would have to change move,remove and other helpers too. Feels unnecessary.